pkg/nodeagent/apis/config/v1alpha1/types.go

```diff
@@ -28,6 +28,10 @@ const (
 	TokenFilePath = CredentialsDir + "/token"
 	// ConfigFilePath is the file path on the worker node that contains the configuration of the gardener-node-agent.
 	ConfigFilePath = BaseDir + "/config.yaml"
+	// KubeconfigFilePath is the file path on the worker node that contains the kubeconfig of the gardener-node-agent
+	KubeconfigFilePath = CredentialsDir + "/kubeconfig"
+	// MachineNameFilePath is the file path on the worker node that contains the machine name where gardener-node-agent runs
+	MachineNameFilePath = CredentialsDir + "/machine-name"
```

Comment:
Why is this in the credentials dir?

Reply:
I put it here to have all credential related files in one directory, because in node-agent context `machine-name` is used as user name.

Reply:
I don't understand. The machine name is not considered secret or confidential. It's confusing to find it in the credentials directory.